### ISSUE TITLE
Safari (Mac + iOS) error on load

### ISSUE THREAD

**Issue by joewiz** [eXistSolutions/hsg-shell#42]
Loading pages like https://history.state.gov/beta/departmenthistory/people/armour-norman from Safari on Mac OS X or iOS yields a pop-up error:

> <img width="435" alt="screen shot 2015-12-08 at 9 53 25 pm" src="https://cloud.githubusercontent.com/assets/59118/11675242/25f3944a-9df6-11e5-9729-3a3017ab8766.png">

The Safari inspector on Mac OS X suggests that app.js is firing a request to frus-ajax.xql:

> https://history.state.gov/beta/exist/apps/hsg-shell/modules/frus-ajax.xql?url=%2Fbeta%2Fdepartmenthistory%2Fpeople%2Farmour-norman

The error message here, naturally, is 

```
<exception>
  <path>/db/apps/hsg-shell/modules/frus-ajax.xql</path>
  <message>
err:XPTY0004 Expected exactly one item for the function to be called, got 0. 
Expression: map:get($config:PUBLICATIONS, untyped-value-check[xs:anyAtomicType, 
$publication-id])?select-section [at line 40, column 13, source: 
/db/apps/hsg-shell/modules/pages.xqm] In function: 
pages:load-xml(xs:string, xs:string, xs:string?, xs:string) 
[40:13:/db/apps/hsg-shell/modules/pages.xqm]
  </message>
</exception>
```

**Comment by joewiz** [comment]
I've confirmed that this network request for frus-ajax.xql is not made on Firefox or Chrome. This issue affects only Safari on Mac OS X and—of more concerning given our focus on enabling mobile access—iOS devices.

**Comment by wolfgangmm** [comment]
I can't see this behaviour on my version of Safari (9.0.1, 10601.2.7.2). Has it been fixed? I do not own any iOS devices, so cannot check there.

**Comment by joewiz** [comment]
Indeed, the problem has vanished. I think I inadvertently fixed this when I removed the `<div id="content-container">` from `pages/departmenthistory/people.html` - see https://github.com/eXistSolutions/hsg-shell/commit/9f6790022893aceaa8523f60d5505dbded197c38#diff-ad04c051f876d78a8c4f45769b4fd34f. I made this change to get rid of the space on either side of the content, since we weren't using ajax prev/next buttons in this page. Something about the presence of that `div/@id` must have caused `app.js` to fire a request to `frus-ajax.xql`. Perhaps Safari does some kind of pre-fetching. In any case, I think this issue is now moot, so I'll close it. If I see this happening elsewhere, I'll report it as a separate issue. Thanks!
